.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - 0.6
-  - 0.8
```

Comment:
After gzip the original file is 1837 bytes. md5.min.js is 2474 after gzip. Webpack really seems to make things worse.